Userland/Libraries/LibGUI/GlyphMapWidget.cpp

```diff
@@ -573,6 +576,8 @@ ErrorOr<void> GlyphMapWidget::set_font(Gfx::Font const& font)
     m_original_font = TRY(font.try_clone());
     m_modified_glyphs.clear();
     AbstractScrollableWidget::set_font(font);
+    did_change_font();
```

Comment:
Is this necessary? Should already be called by `set_font()`

Reply:
Yes it is, if the user of `GlyphMapWidget` doesn't call `set_font()`, `did_change_font()` will never be called.

Reply:
This is in set_font(). It looks like you forgot to delete this when moving it back to the constructor

Reply:
Ahhh yes..